on_the_porch/data_ingestion/ingest_311_911_to_rag.py

```
@@ -0,0 +1,912 @@
+        return 0
+
+    vectordb = _get_vectordb(vectordb_dir)
+    total = 0
+
+    for i in range(0, len(documents), batch_size):
+        batch = documents[i : i + batch_size]
+        try:
+            vectordb.add_documents(batch)
+            total += len(batch)
+            print(f"    → Embedded batch {i // batch_size + 1}: {len(batch)} docs ({total}/{len(documents)} total)")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)